cpp/src/aztec/ecc/curves/bn254/scalar_multiplication/scalar_multiplication.test.cpp

```diff
@@ -18,7 +18,6 @@ namespace {
 auto& engine = numeric::random::get_debug_engine();
```

Comment:
Can we get rid of `srs` as a dependency of `ecc`? Because its used only in the tests.

Reply:
TODO: do this as a separate PR, one idea can be to filter out `io` as its own module for read-write of transcripts. That will help us get rid of the circular dependency.